GoogleForCreators/web-stories-wp
# Improve border-radius mismatch with front-end

## Summary


Converts the border-radius to `%` based on the width and height of the element.
## Relevant Technical Choices



## To-do

- [x] Adjust Karma if needed.



## User-facing changes



## Testing Instructions
- Add an element (test with rectangular shape, text, and media)
- Assign border and border-radius to the element
- Verify the border-radius looks similar in the editor, in the front-end (Preview) and on the thumbnails below the Page.
- Resize the screen/switch devices in the front-end to verify it looks similar in different cases.

Note: Due to the border-width not being responsive, there will be slight differences in radius as well, however, it shouldn't be disturbing/significant.


---



Fixes #5530 
Fixes #5371
